javalin/src/main/java/io/javalin/http/staticfiles/JettyResourceHandler.kt

```diff
@@ -52,6 +59,49 @@ class JettyResourceHandler(val precompressStaticFiles: Boolean = false) : io.jav
         }
     }
 
+    inner class AliasResourceHandler(private val urlPathPrefix: String, private val aliasCheck: AliasCheck) : ResourceHandler() {
```

Comment:
Would it make sense for this to extend `PrefixableHandler` ?

Reply:
I will try